frontend/src/components/charts/Indicator.js

```diff
@@ -7,6 +7,34 @@ import moment from 'moment';
 class Indicator extends Component {
   constructor(props) {
     super(props);
+    this.state = { localComputedTimestamp: null };
+  }
+
+  /**
+   * Updates the local computed timestamp with the one received from the props. Case when it comes from a query via xhr or via WS
+   * @param {props} param0
+   * @returns
+   */
+  static getDerivedStateFromProps({ data: { computedTimestamp } }) {
+    return { localComputedTimestamp: computedTimestamp };
+  }
+
+  /**
+   * @summary sets the local timestamp + 1 second, called on one second interval after the component mounts
+   */
+  updateTimestamp = () => {
+    const { localComputedTimestamp } = this.state;
+    const newTime = moment(localComputedTimestamp)
+      .add(1, 'seconds')
```

Comment:
imho this workaroundish code is not needed and looks ugly.

Reply:
why you say this is workaroundish ? I changed to update the localComputerTimestamp with one min